schema/constants.go

```diff
@@ -36,6 +36,10 @@ const (
 	CreateTableCouponsSQL                   = "create table coupons (slug text, code text, hash text);"
 	CreateIndexCouponsSQL                   = "create index index_coupons on coupons (slug);"
 	CreateTableModeratedStoresSQL           = "create table moderatedstores (peerID text primary key not null);"
+	CreateMessagesSQL                       = "create table messages (messageID text primary key not null, orderID text, message_type integer, message blob, peerID text, url text, acknowledged bool, tries integer, created_at integer, updated_at integer);"
+	CreateIndexMessagesSQL1                 = "create index index_messages1 on messages (messageID);"
+	CreateIndexMessagesSQL2                 = "create index index_messages2 on messages (orderID, message_type);"
+	CreateIndexMessagesSQL3                 = "create index index_messages2 on messages (peerID, message_type);"
```

Comment:
I don't see a lookup which uses `CreateIndexMessagesSQL1` or `CreateIndexMessagesSQL3`. Remove?

Reply:
If `CreateIndexMessagesSQL3` is not removed, please update the index name so it's not repeating the same from `CreateIndexMessagesSQL2`